### DISCUSSION TITLE
Blueretro.io goes down. How to configure my devices?

### DISCUSSION THREAD

**Discussion by golimonkey** [darthcloud/BlueRetro#1014] [Q&A]
Is there a way to configure blueretro devices while offline?

**Comment by zeus86** [comment]
I guess then you grab the code from https://github.com/darthcloud/BlueRetroWebCfg and spawn up your own webserver until someone builds a standalone app.

**Comment by darthcloud** [comment]
You could run it locally but it's a pain.

The easiest way is to fork it on GitHub and enable github-page to have your own copy.

**Reply by zeus86** [reply]
I'm a fulltime linux-admin in my dayjob, and own some domains and static ip-addresses, so I would run it locally, but yes, for average joe github-/gitlab-pages are most likely the superior option :)
